pfcli/main.py

```diff
@@ -49,6 +49,25 @@
     headers=["Name", "Username", "Email"]
 )
 
+@app.command(help="Sign up to PerfiFlow")
+def signup(
+    username: str = typer.Option(..., prompt="Enter Username"),
+    name: str = typer.Option(..., prompt="Enter Name"),
+    email: str = typer.Option(..., prompt="Enter Email"),
+    password: str = typer.Option(..., prompt="Enter Password", hide_input=True)
```

Comment:
Please make it re-enter password for confirmantion.

e.g., "Enter password" -> "Enter the password again"